### DISCUSSION TITLE
Uninstall instructions

### DISCUSSION THREAD

**Discussion by ostermayer** [minibolt-guide/minibolt#12] [Ideas]
Scripts for uninstall would be helpful for users that transition from one service to another. Example have electrs but want uninstall to try fulcrum

**Comment by twofaktor** [comment]
Regarding the script concept, the same answer as #2

Anyway, there are some guides that don't include the uninstall section yet, in the roadmap, there is the uninstall section in the Electrs guide.

Thanks!
